pwd/metadata.txt

```diff
@@ -0,0 +1,5 @@
+srctype = micropython-lib
+type = module
+version = 0.0.1
```

Comment:
Version should be 0.1, 0.0.x is for completely dummy modules.

Reply:
ok